src/core/ext/transport/chttp2/transport/legacy_frame.h

```diff
@@ -33,6 +33,7 @@ typedef struct grpc_chttp2_transport grpc_chttp2_transport;
 #define GRPC_CHTTP2_FRAME_PING 6
 #define GRPC_CHTTP2_FRAME_GOAWAY 7
 #define GRPC_CHTTP2_FRAME_WINDOW_UPDATE 8
+#define GRPC_CHTTP2_FRAME_SECURE 200
```

Comment:
s/SECURE/SECURITY - here and everywhere I think (justification in chttp2_transport.cc)

Reply:
Done.